### ISSUE TITLE
Would it be possible to add support for Spanish and Italian languages?

### ISSUE THREAD

**Issue by johnfelipe** [HarryDulaney/pfolie#95]
If you share strings in any format, such as po, JSON, XML, or something else, I will translate them to contribute and include them in this app's i18n multilingual support.

**Comment by HarryDulaney** [comment]
Yes definitely, I definitely plan to add multi-language support and we will include Spanish and Italian for sure!

**Comment by johnfelipe** [comment]
when you have strings in any format pls share and will collaborate
